labs-ios-starter/View Controllers/Impact Statistics/Production Reports/ExistingReportViewController.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  ExistingReportViewController.swift
+//  labs-ios-starter
+//
+//  Created by Joseph Veverka on 10/7/20.
+//  Copyright © 2020 Spencer Curtis. All rights reserved.
+//
+
+import UIKit
+
+class ExistingReportViewController: UIViewController {
+    
+    //MARK: - IBOutlets
+
+    @IBOutlet var reportDateLabel: UILabel!
+    @IBOutlet var hoursWorkedLabel: UILabel!
+    @IBOutlet var soapmakersWorkedLabel: UILabel!
+    @IBOutlet var barsProducedLabel: UILabel!
+    @IBOutlet var addAPhotoLabel: UILabel!
+    @IBOutlet var reportImages: UIImageView!
+    @IBOutlet var deleteButtonOutlet: UIButton!
+    @IBOutlet var editButtonOutlet: UIButton!
+    
```

Comment:
Create another Mark for View Life Cycle, so it does not seem that the ViewDidLoad is part of the IBOutlets.